catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/transformer/TransformerManager.java

```diff
@@ -11,10 +11,12 @@
  * is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.spatial.ogc.csw.catalog.transformer;
+package org.codice.ddf.spatial.ogc.csw.catalog.common.transformer;
```

Comment:
So nothing was embedding spatial-csw-transformer before this move that needs to update to no longer embed it?